website/docs/from_provider/provider_vs_riverpod.mdx

```diff
@@ -191,7 +131,7 @@ Inside the `build` method, use "watch". Inside click handlers and other events,
 use "read".
 :::
 
-### Reading providers: Consumer
+## Reading providers: Consumer
```

Comment:
I think it's be worth talking about Consumer2 vs calling ref.watch multiple times